### PR TITLE
Fix tmp dir creation

### PR DESCRIPTION
**Pull request by bdavs3**
On my machine, e.g., which did not have a /tmp directory, `mkdirSync('/tmp/slsk')` throws an error. By adding the recursive flag, the base directory will be created if it does not exist.

| Q                           | A
| --------------------------- | ---
| Bug fix                     | true
| New feature                 | false
| Deprecations                | false
| Tests pass (please, really) | false, however there were 3 failing tests before I made changes, and I did not address them
| Fixed issues                | N/A

### CODE REVIEW

**Comment by ndr-brt** [comment]
@bdavs3 I know, some years passed, but could you fix the conflicts, then I could proceed with review & merge. thanks

**Comment by bdavs3** [comment]
@ndr-brt all done!

**Comment by bdavs3** [comment]
Oops, I was taking the wrong changes, so I just added one more commit

File: lib/peer/distributed-peer.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ const Messages = require('../messages.js')
 const MessageFactory = require('../message-factory.js')
 const debug = require('debug')('slsk:peer:distributed:i')
 
+let stack = require('../stack')
```
Comment (ndr-brt) [inline lib/peer/distributed-peer.js:6]: is this needed?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "slsk-client",
-  "version": "1.0.13",
+  "version": "1.1.0",
```
Comment (ndr-brt) [inline package.json:3]: I think the version shouldn't be changed from a PR

**Review by ndr-brt — CHANGES_REQUESTED** [review]

File: lib/peer/distributed-peer.js (hunk 1 above)
Comment (ndr-brt) [reply lib/peer/distributed-peer.js:6]: I mean, I was curious about this addition, but in fact seems to be already defined in master branch, maybe it's just a github flaw

Comment (bdavs3) [reply lib/peer/distributed-peer.js:6]: Interesting. I see that as well. I suppose it should be left in this PR though, otherwise it'll get taken out.

**Comment by bdavs3** [comment]
@ndr-brt I saw your comment about versioning. Did you delete it? I believe the updated version is needed for the recursive `mkdir` call.

**Review by ndr-brt — APPROVED** [review]
Yes, I removed it because I noticed that the latest released version is 1.1.0, so maybe it didn't got updated

**Review by ndr-brt — APPROVED** [review]
Looks good!

**Pull request merged**